workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/container/fixtures/BackStackContainerLifecycleActivity.kt

```diff
@@ -40,7 +40,7 @@ internal class BackStackContainerLifecycleActivity : AbstractLifecycleTestActivi
       context: Context,
       container: ViewGroup?
     ): ScreenViewHolder<BaseRendering> =
-      ScreenViewHolder(initialEnvironment, View(context)) { _, _ -> /* Noop */ }
+      ScreenViewHolder(initialEnvironment, View(context)) { _, _ -> }
```

Comment:
I don't get these? Did ktlint remove these now?

Reply:
https://github.com/pinterest/ktlint/issues/1980

For the KtLint `0.49.x` season, the `comment-wrapping` rule doesn't allow for single-line block comments.  This is extracted into its own rule for `0.50.0`.